XIPs/xip-43-permission-pref-proofs.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Permission preference proofs
+description: Enables recipients to sign to specify inbox permission preferences
+author: @rygine @neekolas @saulmc
+discussions-to: https://community.xmtp.org/t/xip-43-permission-preference-proofs/552
+status: Draft
+type: Standards
+category: Interface
+created: 2024-02-22
+---
+
+## Abstract
+
+This XIP proposes to add an _optional_ signed payload to conversations that client apps can consider as proof that a recipient has granted a sender permission to reach their main inbox.
+
+## Motivation
+
+Today, a recipient must use a full XMTP client to set a universal permission preference such as allowing a sender. This presents challenges to senders due to the XMTP client's bundle size and integration requirements. For integrations such as simple subscribe buttons, these hurdles can be blockers.
+
+To solve these issues, senders can simply ask users to produce a signature attesting to the permission preference update. Inbox apps can consider this signature as proof that the recipient has explicitly opted in to receive the sender's messages. This eliminates the bundle size problem, as very little code is required to initiate signing a message with a user's wallet. It also greatly simplifies integration by providing a single function to return a signed message.
```

Comment:
the last sentence was updated on discourse to this: `It also greatly simplifies integration by providing a single function to obtain the user’s signature and return an encoded payload to be used by SDKs to verify the granted permission.`